packages/repository/src/repository-mixin.ts

```diff
@@ -62,6 +62,18 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
         .tag('repository');
     }
 
+    /**
+     * Retrieve the repository instance from the given Repository class
+     *
+     * @param repo The repository class to retrieve the instance of
+     */
+    // tslint:disable-next-line:no-any
+    async getRepository<Repo extends Repository<any>>(
```

Comment:
Did you consider using `R` instead of `Repo` for the generic variable? I went looking for the `Repo` type not realizing it was a generic like `T`. 

-- Maybe it's just me, but I expect to see generics as one capital letter. Usually `T` but that's taken in this case so `R` might make sense.

Reply:
Yeah I'll take a look around in the codebase to see how people named their generics.